### ISSUE TITLE
Nic + Eva Meeting 2023-03-27

### ISSUE THREAD

**Issue by evamaxfield** [PugetSoundClinic-PIT/ProjectTracking#106]
# Major Projects and Deliverables Due Soon

* PIT UN Case
  * Had any time to review what I already have?
  * If not, I can just take it an run with it
* JOSE Paper
  * Still need to work on building out the content of the repo
  * Building out the content will also naturally be doing the JASIST analysis
  * Hopefully this is a good resource for the master's student who is starting in May on School Boards
* Gig Plat
  * Analysis review as a team -- ASAP
  * Big point of discussion there is "Multinomial Logit" vs "Series of Binary Logit" vs "sklearn predictive model and investigation by weights"
  * Still need to make the timeline plots and normalization table via King County reported counts
* JASIST Paper (or larger venue?)
  * Work for now will be doubled up one JOSE Paper

## Priority of Major Projects

1. PIT UN Case
2. Gig Plat Analysis
3. JOSE Paper
4. JASIST Paper

# Other Projects and Courses

* EAGER
  * AUGUR querying? What do we want to do with it?
  * Is there a next deliverable on that?
* Campaign Access Eval
  * sentiment analysis data returned to Cici. Should I do something w/ that or let her take the lead?
* Computing for Social Good
  * Can swap for Emma Spiro's class
  * From looking at prior syllabus the project I am interested in comes from discussions at the conf -- the "Train My AI Navigator" from Ben Lee but applied to CDP data w/ an extension which is after you train and are happy with the selected data, place the found content on a timeline.
* Research Practicum II w/ Bill
  * pinged again, if no response by tomorrow I think I need to do something else
  * Also not sure how to phrase "explore this area" as a research question??

# Notes from Conf

## Highlights
* People were overall really impressed with what we have built
* I got to deploy an instance for a journalist in less than 24 hours turnaround
  * Their use case was trying to follow police union (and chief) testimony to council over a two year period. Something w/ the chief is the focus.
* Dustin, Serder, and Ernest, were all great. We all want to collaborate.
* Knight Foundation rep was there specifically to look for ways to fund collaborative efforts.
  * Someone mentioned "Ford Foundation Public Interest Tech" funding as a way to get more "makers" into this work, I said that is how our lab is funded but funding is soon up.
  * Similarly, I specifically prodded Big Local News and Dustin and Ernest as to their thoughts on collab -- in general the easiest target is data interop, and shared infra / microservices.
* I mentioned my concern about how unarchived all of this data is, relating to interop, there is no standard format and no shared archive for all of us. What happens when one of the projects dies, does the data we have compiled also die? Ben Walsh sent an email to archive.org asking about it and we instantly got a response back and they are good to host our data if we want to backup.

## Observations
* More journalist heavy than expected but that was nice honestly
* Some people are literally summarizing policy with ChatGPT

## Collabs
* Darla Cameron from Texas Tribune is interested in School Board stuff
* AgendaWatch + Minutes + CDP data interop, shared microservices, shared archives
* Alexander Spangher -- Computational approaches to "detecting newsworthiness" -- working with BLN
* Justin Grimer out of Stanford Poli Sci (recommendation from someone else)
* State Deployments: https://iatpp.calpoly.edu/digital-democracy

## Research
* Documenteers data might be good for training certain models
* Local Interest Groups
  * take in volume of speaker as a weight to logit models
  * can we measure what types of comments swap certain council members (budgetary reasons, appeal to emotion, appeal to something else?)
  * if we have logit models for each city council we study, which cities have the most equal weights and which have the least equal weights (between pro and con) -- what makes those cities different, or city populations, or public comments?
* Segmentation models should be trained as span-cat models, certain sections of the meeting can be reused. In certain councils, there are multiple different periods of public comment.

## Engineering and Feature Requests
* RegEx Search in transcripts (already in progress via Alon [Shak's brother])
* Annotation area in the transcripts + CSV export or some other format export of annotations
* MAYBE deploy datasette w/ every instance
* Push audio file to podcast feed (especially if we can segment)
* Journalists like trackers (not dashboards) but services that can ping you
* Multi city search
* Reduce / get rid of junk minutes items before ingestion via ChatGPT "what are the line items from this list"

## Best Idea

Train Your AI Navigator for CDP

* as David Kroman told us years ago, "if i need a quote, i can usually just call someone"
* others agreed with that
* BUT there are pieces that are done that we can help with, specifically the ones which track statements over a long period of time -- multiple journalists talked about this. it isn't a large part of their job but it is a small part that is incredibly time intensive (and boring)
* annotate parts of a transcript with what you are interested in, get presented with similar, provide positive and negative annotations to a few other, keep narrowing in on a topic, then present all these snippets on a timeline

can introduce all sorts of features here, the "sentiment over time" is easy to add to such a timeline for example

**Comment by evamaxfield** [comment]
* Send email on AUGUR
* Give Cici directions on plotting the sentiment data
